reference-implementation/web-platform-tests/readable-streams/general.js

```diff
@@ -608,6 +608,51 @@ promise_test(() => {
 
 }, 'ReadableStream pull should be able to close a stream.');
 
+promise_test(() => {
+
+  let pullCalled = false;
+
+  const rs = new ReadableStream({
+    pull(c) {
+      pullCalled = true;
+      c.error();
+    }
+  });
+
+  const reader = rs.getReader();
+  return reader.closed.then(() => {
+    assert_unreached('closed should be rejected');
+  }, e => {
+    assert_true(pullCalled);
+  });
+
+}, 'ReadableStream pull should be able to error a stream.');
+
+promise_test(() => {
+
+  let pullCalled = false;
+
+  const theError = new Error('rejected!');
+  const otherError = new Error('other!');
+
+  const rs = new ReadableStream({
+    pull(c) {
+      pullCalled = true;
+      c.error(theError);
+      throw theError;
```

Comment:
should be otherError, right? And probably want to use `{ name: 'rejected!' }`, `{ name: 'other' }` so that you can use `promise_rejects` and stuff. See https://github.com/whatwg/streams/commit/7f19175415d9b57af04cb8c2cd355ca38101f0aa